src/css/MoviePoster.css

```diff
@@ -1,6 +1,13 @@
 .movie-poster {
   width: auto;
   height: 60vh;
+  cursor: pointer;
+  transition-duration: 0.5s;
+  margin-bottom: 50px;
+}
+
+.movie-poster:hover {
```

Comment:
Nice; I'm a big fan of the hover-transform. Makes the whole thing look more dynamic, y'know?